### PR TITLE
Adding is file test for validate

### PR DESCRIPTION
**Pull request by esharf**
## Status
Ready.

## Related Issues
fixes: https://github.com/demisto/etc/issues/21617
fixes: https://github.com/demisto/etc/issues/21421

## Description
Added a check for the files.

## Screenshots
before
before
after
after

### CODE REVIEW

**Comment by Itay4** [comment]
@esharf @bakatzir ?

File: demisto_sdk/commands/validate/file_validator.py [hunk 1]
```diff
@@ -130,6 +130,9 @@ def get_modified_files(files_string, tag='master', print_ignored_files=False):
                 file_status = 'r'
                 file_path = file_data[2]
 
+            if not os.path.isfile(file_path):
```
Comment (yuvalbenshalom) [inline demisto_sdk/commands/validate/file_validator.py]: how is this tested? what is the reason that a file will not have a path? I am not sure about this.

**Comment by yaakovi** [comment]
@yuvalbenshalom @bakatzir are you reviewing this or should i?

**Comment by bakatzir** [comment]
I do not.

**Comment by bakatzir** [comment]
@esharf Please add the fixes to the CHANGELOG before merging.

**Comment by roysagi** [comment]
@esharf Changelog is good.

**Pull request merged**